### DISCUSSION TITLE
Folder "table_plots" missing

### DISCUSSION THREAD

**Discussion by Better-L** [ep142/FoodMicrobionet#2] [Ideas]
Hi,
I want to advise that the folder "table_plots" is missing in the repository path "FoodMicrobionet-master\shiny_apps\shiny_FMBN_2_4_1\ShinyFMBN2_4_1\output".

All the best,
Luca

**Comment by ep142** [comment]
Thank you, I will fix it next week

Il giorno mer 16 mar 2022 alle 14:58 LukaBetter ***@***.***>
ha scritto:

> Hi,
> I want to advise that the folder "table_plots" is missing in the
> repository path
> "FoodMicrobionet-master\shiny_apps\shiny_FMBN_2_4_1\ShinyFMBN2_4_1\output".
>
> All the best,
> Luca
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ep142/FoodMicrobionet/discussions/2>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGWCW3BPS4YIAJETNOGPGLDVAHSJLANCNFSM5Q37DTMA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Prof. Eugenio Parente
Scuola di Scienze Agrarie, Forestali, Alimentari ed Ambientali - Università
degli Studi della Basilicata
Potenza, Italy

**Reply by ep142** [reply]
OK, apparently GitHub did not like the name of the folder (the underscore?). I have changed it and changed the app...

**Reply by Better-L** [reply]
Thank you!

**Reply by ep142** [reply]
even worse! GitHub is so persnickety that I was unable to create an empty folder and upload it
